addons/edi/views/edi_document_views.xml

```diff
@@ -62,8 +62,11 @@
 	    <div class="oe_title">
 	      <label for="name" class="oe_edit_only"/>
 	      <h1>
-		<field name="name" placeholder="e.g. PMD00001"/>
+					<field name="name" placeholder="e.g. PMD00001"/>
```

Comment:
Nit: spacing looks off here.

Reply:
TBH the indentation is not well on original file, there are spaces and tabs mixed in original file